src/main/java/com/yandex/money/api/methods/AccountInfo.java

```diff
@@ -264,6 +276,15 @@ public Builder setYandexMoneyCards(List<YandexMoneyCard> yandexMoneyCards) {
         }
 
         /**
+         * @param virtualCards list of Yandex.Money virtual cards
+         * @return itself
+         */
+        public Builder setVirtualCards(List<YandexMoneyCard> virtualCards) {
+            this.virtualCards = checkNotNull(virtualCards, "virtualCards");
```

Comment:
It is better to skip the assignment if parameter is ```null```, because the field has default value.